membership-attribute-service/app/models/Attributes.scala

```diff
@@ -41,7 +42,8 @@ case class Attributes(
   lazy val isPatronTier = Tier.exists(_.equalsIgnoreCase("patron"))
   lazy val isStaffTier = Tier.exists(_.equalsIgnoreCase("staff"))
   lazy val isPaidTier = isSupporterTier || isPartnerTier || isPatronTier || isStaffTier
-  lazy val isContributor = RecurringContributionPaymentPlan.isDefined
+  //TODO - only use OneOffContributionDate for verified users
+  lazy val isContributor = RecurringContributionPaymentPlan.isDefined || OneOffContributionDate.isDefined
```

Comment:
I think we should separate out one-offs and recurring here

Reply:
good point, `isContributor` is used for more than just setting `showSupportMessaging`

Reply:
in fact, further down, this variable passed as to `ContentAccess` as `recurringContributor`